src/renderer/screens/Editor/Editor.js

```diff
@@ -257,36 +336,67 @@ const Editor = (props) => {
   const KeymapSVG = focus.focusDeviceDescriptor.components.keymap;
 
   let title;
-  if (hasColormap() && hasKeymap()) {
+  if (openMacroEditor) {
+    title = t("app.menu.macroEditor");
+  } else if (hasColormap() && hasKeymap()) {
     title = t("app.menu.editor");
   } else if (hasKeymap()) {
     title = t("app.menu.layoutEditor");
   } else {
     title = t("app.menu.colormapEditor");
   }
 
+  const currentKey =
+    selectorKey || keymap.custom[currentLayer][currentKeyIndex];
+
+  let mainWidget;
+  if (openMacroEditor) {
```

Comment:
Toggling the main widget like this feels kind of weird to me, but I can't put my finger on why.

Reply:
It certainly isn't the cleanest thing. For now, I'd leave it as-is, and when I do a Suspense + Skeleton pass over all of Chrysalis, I'll clean this (and similar patterns elsewhere, too) up. The whole of Editor could use some refactoring, but I feel that's best done separately.

Reply:
ok